dev-util/clazy/clazy-1.10-r1.ebuild

```diff
@@ -3,7 +3,9 @@
 
 EAPI=7
 
-inherit cmake
+LLVM_MAX_SLOT=12
```

Comment:
Are you sure? As far as I am aware 1.10 had a commit for LLVM 13 compatibility and I had no problems.